src/System.Private.ServiceModel/System.Private.ServiceModel.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
```

Comment:
Reviewers: the update the SLN was necessary to accommodate the prior PR that moved the Infrastructure source project.  As a consequence of that update, VS touched several .csproj's to fix spacing and GUID's.  I prefer to take those touches so we don't see them again.